### ISSUE TITLE
[UI] : Cannot view istio configs

### ISSUE THREAD

**Issue by draeron** [kiali/kiali#619]
**Describe the bug**

![image](https://user-images.githubusercontent.com/2256246/47884753-6925f500-de08-11e8-8432-340aac99fb62.png)

After investigation, it seems the problems is with the react components, i get this in mozilla's console : 

```
TypeError: "n.state.istioObjectDetails.permissions is undefined"
	renderRightToolbar https://istio-kiali.stingray-sandbox.com/static/js/main.09455b18.js:1:2316271
	renderEditor https://istio-kiali.stingray-sandbox.com/static/js/main.09455b18.js:1:2315899
	render https://istio-kiali.stingray-sandbox.com/static/js/main.09455b18.js:1:2319436
	Ki https://istio-kiali.stingray-sandbox.com/static/js/main.09455b18.js:1:1230016
	Yi https://istio-kiali.stingray-sandbox.com/static/js/main.09455b18.js:1:1229859
	eo https://istio-kiali.stingray-sandbox.com/static/js/main.09455b18.js:1:1232493
	Po https://istio-kiali.stingray-sandbox.com/static/js/main.09455b18.js:1:1246553
	Do https://istio-kiali.stingray-sandbox.com/static/js/main.09455b18.js:1:1246899
	ha https://istio-kiali.stingray-sandbox.com/static/js/main.09455b18.js:1:1251571
	ca https://istio-kiali.stingray-sandbox.com/static/js/main.09455b18.js:1:1251225
	Io https://istio-kiali.stingray-sandbox.com/static/js/main.09455b18.js:1:1249737
	enqueueSetState https://istio-kiali.stingray-sandbox.com/static/js/main.09455b18.js:1:1218571
	setState https://istio-kiali.stingray-sandbox.com/static/js/main.09455b18.js:1:1167827
	fetchIstioObjectDetailsFromProps https://istio-kiali.stingray-sandbox.com/static/js/main.09455b18.js:1:2314216
 react-dom.production.min.js:171:139
```

**Versions used**
docker image : `docker.io/kiali/kiali:v0.9`

Content from ui's about box: 
```
kiali-ui 0.10.0 (b017ede0af1893c5215ecfa350c15de3d0b9c636)
kiali v0.9.0 (b11bc05013df2457e479eb381c52e924e27db19d)
Istio 1.0.3
Prometheus 2.3.1
Kubernetes v1.10.5 (kube-aws)
```

**To Reproduce**
kiali was installed through istio's 1.0.3 chart. Some modification was made because it was missing some rbac permissions but it's pretty much vanilla otherwise.

**Comment by pilhuhn** [comment]
Recorded in Jira as https://issues.jboss.org/browse/KIALI-1859

**Comment by pilhuhn** [comment]
@draeron Can you please have a look at the logs of the Kiali pod/server container and see if there are also error messages there? If so can you please paste them?

**Comment by jon-walton** [comment]
I'm also seeing this with the same versions as above.

also, a much easier to digest stacktrace from chrome

```
react-dom.production.min.js:171 TypeError: Cannot read property 'delete' of undefined
    at t.n.renderRightToolbar (IstioConfigDetailsPage.tsx:222)
    at t.n.renderEditor (IstioConfigDetailsPage.tsx:198)
    at t.render (IstioConfigDetailsPage.tsx:311)
    at Ki (react-dom.production.min.js:156)
    at Yi (react-dom.production.min.js:155)
    at eo (react-dom.production.min.js:162)
    at Po (react-dom.production.min.js:192)
    at Do (react-dom.production.min.js:193)
    at ha (react-dom.production.min.js:205)
    at ca (react-dom.production.min.js:204)
```

**Comment by jmazzitelli** [comment]
That last v0.9 build looks to be bad. I'll take a look today. Someone else saw something like this yesterday. We need to look closer at our build process.

**Comment by draeron** [comment]
i have switched to `v0.9.0` and it works, so this means the default in the istio chart is not the good one. are you guys the maintainers?

**Comment by jmazzitelli** [comment]
Yes, @draeron  we are maintaining it - going to hopefully address this today. Thanks for bringing it up.

**Comment by pilhuhn** [comment]
This should have been fixed in version "v0.9.1.helmfix" and (@jmazzitelli correct me if I'm wrong) also 0.10.0

**Comment by pilhuhn** [comment]
Closing as I think it is fixed. Please re-open (or a new one if now)
